tfhe/src/high_level_api/booleans/mod.rs

```diff
@@ -159,7 +195,7 @@ impl FheTryEncrypt<bool, CompressedPublicKey> for crate::FheBool {
     fn try_encrypt(value: bool, key: &CompressedPublicKey) -> Result<Self, Self::Error> {
         let key = &key.key;
         let ciphertext = key.key.encrypt(u64::from(value));
-        Ok(Self::new(ciphertext))
+        Ok(Self::new(BooleanBlock::new_unchecked(ciphertext)))
```

Comment:
internally those ops still return a shortint ciphertext ?

Reply:
Hum, Yeah I haven't added encryt_bool for the different PublicKey types of integer. Should I add them in this PR ?

Reply:
is it a large change ? otherwise it's fine to put it in a separate PR I would say

Reply:
I added the 2 missing functions